src/RecoveryServices/RecoveryServices.Backup/Cmdlets/Restore/GetAzureRmRecoveryServicesBackupWorkloadRecoveryConfig.cs

```diff
@@ -218,22 +225,44 @@ public override void ExecuteCmdlet()
                             ((AzureWorkloadProtectableItem)TargetItem).ServerName) == 0)
                         {
                             string itemId = GetItemId(RecoveryPoint.Id);
-                            IList<SQLDataDirectory> dataDirectoryPaths = GetRpDetails(vaultName, resourceGroupName);
-                            foreach (var dataDirectoryPath in dataDirectoryPaths)
+
+                            if(UseSecondaryRegion)
```

Comment:
Is it possible to add a test case for this cmdlet update?

Reply:
It is slightly tricky to add the test case for this. I'm thinking for the same. I'll let you know by EOD ? will it be ok ?

Reply:
It is OK. Thank you!

Reply:
I've added the test case, pls check now!